pinc/release_queue.inc

```diff
@@ -1,8 +1,13 @@
 <?php
 // A place for shared functions dealing with release queues.
 
+include_once($relPath.'misc.inc');
+include_once($relPath.'user_is.inc');
+
 $today_MMDD = date('md', strtotime('today'));
 $tomorrow_MMDD = date('md', strtotime('tomorrow'));
+$user_can_see_queue_settings = $ordinary_users_can_see_queue_settings ||
```

Comment:
If you want this to be really global, we need to include `global $user_can_see_queue_settings;` before this. Otherwise it could be inadvertently not global if this file is included inside another function, for instance.

I don't love that we're shoving another global out there but we're just moving it around from where it was before I suppose. Ideally this would be something like:
```diff
--- a/pinc/User.inc
+++ b/pinc/User.inc
@@ -152,6 +152,8 @@ class User

     public function __get($name)
     {
+        global $ordinary_users_can_see_queue_settings;
+
         switch ($name) {
             case "profile":
                 if ($this->current_profile) {
@@ -191,6 +193,9 @@ class User
                 }

                 return $this->current_profile;
+            case "can_see_queue_settings":
+                return $ordinary_users_can_see_queue_settings ||
+                    user_is_a_sitemanager() || user_is_proj_facilitator();
             default:
                 return $this->table_row[$name];
         }
```
and then here:
```php
User::load_current()->can_see_queue_settings;
```

Reply:
OK. I've moved it to a free function in user_is.inc, since it already relies on functions from that file.